dusty/compiler/compose/helper_functions.py

```diff
@@ -0,0 +1,52 @@
+from ...source import Repo
```

Comment:
Can we call this module `common` instead of `helper_functions`? That's really what these are, right, functions that need to be imported in multiple modules so we put them here to keep our imports clean?